doc/api/cli.md

```diff
@@ -1602,6 +1614,12 @@ and the line lengths of the source file (in the key `lineLengths`).
 }
 ```
 
+### `NO_COLOR=1`
+
+The [`NO_COLOR`][] environment variable is an informal standard supported by a
+growing number of applications for disabling ANSI color escape codes. Within
+Node.js, `NO_COLOR` is an alias for `NODE_DISABLE_COLORS`.
```

Comment:
Re-suggesting this optional nit in case it got lost before when I made two conflicting suggestions. I don't think the first sentence is necessary, or at least shouldn't be what we start with. 

```suggestion
[`NO_COLOR`][]  is an alias for `NODE_DISABLE_COLORS`.
```